tests/mutation_reader_test.cc

```diff
@@ -1079,7 +1079,7 @@ class dummy_file_impl : public file_impl {
 
 SEASTAR_TEST_CASE(reader_restriction_file_tracking) {
     return async([&] {
-        reader_concurrency_semaphore semaphore(100, 4 * 1024);
+        reader_concurrency_semaphore semaphore(100, 4 * 1024, "reader_restriction_file_tracking");
```

Comment:
hint: `get_name()` will return the name of the test.

Reply:
Good one! Done.